rsts/user/features/single_task_execution.rst

```diff
@@ -0,0 +1,90 @@
+.. _features-singletaskexec:
+
+Single Task Excution
+####################
+
+What are single task executions?
+================================
+
+Tasks are the most atomic unit of execution in Flyte.  Although workflows are traditionally composed of multiple tasks with dependencies
+defined by shared inputs and outputs it can be helpful to execute a single task during the process of iterating on its definition.
+It can be tedious to write a new workflow definition every time you want to excecute a single task under development but single task
+executions can be used to easily iterate on task logic.
+
+Launch a single task
+====================
+
+After you've built an image with your updated task code, create an execution using launch:
+
+.. code-block:: python
+
+     @inputs(plant=Types.String)
+     @outputs(out=Types.String)
+     @python_task
+     def my_task(wf_params, plant, out)
+         ...
+
+
+     my_single_task_execution = my_task.launch(project="my_flyte_projext", domain="development", inputs={'plant': 'ficus'})
+     print("Created {}".format(my_single_task_execution.id))
+
+Just like workflow executions, you can optionally pass a user-defined name, labels, annotations, and/or notifications when launching a single task.
+
+The type of ``my_single_task_execution`` is `SdkWorkflowExecution <https://github.com/lyft/flytekit/blob/1926b1285591ae941d7fc9bd4c2e4391c5c1b21b/flytekit/common/workflow_execution.py#L14>`_
+and has the full set of methods and functionality available for conventional WorkflowExecutions.
+
+
+Fetch and launch a single task
+==============================
+
+Single task executions aren't limited to just tasks you've defined in your code. You can reference previously registered tasks and launch a single task execution like so:
+
+.. code-block:: python
+
+     from flytekit.common.tasks import task as _task
+
+     my_task = _task.SdkTask.fetch("my_flyte_project", "production", "workflows.my_task", "abc123")  # project, domain, name, version
+
+     my_task_exec = my_task.launch(project="my_other_project", domain="development", inputs={'plant': 'philodendron'})
+     my_task_exec.wait_for_completion()
```

Comment:
Can Admin withstand potentially tens of thousands of these polling?

Reply:
Do you think we should move to websockets?

Reply:
Following up offline

Reply:
Why websockets? These should be converted to long poll if needed. But we don’t need it at all I feel. This is a usecase only for testing